File: backends/php-laravel/app/Domain/Invitation/Entities/Invitation.php
<?php

/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */

declare(strict_types=1);

namespace App\Domain\Invitation\Entities;

use App\Support\Entities\AbstractEntity;

/**
 * 招待Entityクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 * @package App\Domain\Invitation\Entities
 */
class Invitation extends AbstractEntity
{
    public ?int $id = null;

    public ?string $token = null;

    /**
     * 招待用の完全 URL（API 応答・コピー用）。
     */
    public ?string $url = null;

    /**
     * 表示用の省略 URL（長いトークン部分を ... で省略）。
     */
    public ?string $displayUrl = null;
}

File: backends/java-spring/src/main/java/com/authorization/domain/invitation/entities/Invitation.java
/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */
package com.authorization.domain.invitation.entities;

import com.authorization.support.entities.AbstractEntity;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * 招待Entityクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
@Data
@EqualsAndHashCode(callSuper = true)
public class Invitation extends AbstractEntity {

    private Long id;
    private String token;
    private Integer role;
}
